crowbar_framework/app/models/proposal_queue.rb

```diff
@@ -0,0 +1,35 @@
+class ProposalQueue < ActiveRecord::Base
+  # FIXME: remove this when the export is properly implemented
+  class_attribute :chef_type
+
+  self.chef_type = "data_bag_item"
+  # only accept one default record named "queue"
+  self.primary_key = "name"
+
+  serialize :properties, Utils::JSONWithIndifferentAccess
+
+  def proposals
+    ProposalQueue.find("queue").properties["proposal_queue"]
+  end
+
+  def <<(item)
+    pq = ProposalQueue.find("queue")
```

Comment:
I would prefer using `ProposalQueue` over any of the `self` constructs, because it is more explicit. Repetition of the class name I wouldn't consider as an issue, because somehow you need to identify which class to call.

I would be concerned about the repetition of the `find` argument, because this looks like a potential source of errors, e.g. when mistyping "queue". So wrapping this in a method as proposed sounds like the right thing to me.